migrations/20181206104443_logs.js

```diff
@@ -1,8 +1,8 @@
+exports.up = knex =>
+  knex.schema.createTable('logs', t => {
+    t.increments('id').primary()
+    t.integer('user_id').unique()
```

Comment:
Need to change this to:
t.integer('user_id').references(users.id')

Reply:
Thanks Chris :) I fixed all FKs